tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -51,6 +51,16 @@ func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRendere
 			return intValueRenderer{}, nil
 		}
 
+	case fd.Kind() == protoreflect.MessageKind:
+		md := fd.Message()
+		fullName := md.FullName()
+		switch fullName {
+		case "google.protobuf.Timestamp":
+			return NewTimestampValueRenderer(), nil
+		}
+		// TODO default message renderer
+		return nil, fmt.Errorf("no value renderer for message %s", fullName)
```

Comment:
Instead of hardcoding this switch case, I propose to use a registry, see how it's done for `scalars`.

Then, in the Textual struct's `init` function, we can initialize the registry to `"google.protobuf.Timestamp"->NewTimestampValueRenderer()`

Reply:
Done.